.github/workflows/stale.yml

```diff
@@ -0,0 +1,34 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 11 * * 1-4"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/stale@v3
+      with:
+        operations-per-run: 1000
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I made DEVINF-334 to tracking creating this.

Reply:
Turns out we didn't need this, as it's auto-generated. See:  https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret